app.yaml

```diff
@@ -15,34 +15,24 @@ inbound_services:
 
 handlers:
 - url: /favicon.ico
-  static_files: static/images/favicon.ico
-  upload: static/images/favicon.ico
+  static_files: static/common/favicon.ico
+  upload: static/common/favicon.ico
   secure: always
   http_headers:
     Cache-Control: 'public, max-age=2592000'
     Vary: Accept-Encoding
-- url: /images
-  static_dir: static/images
-  secure: always
-  http_headers:
-    Cache-Control: 'public, max-age=600'
-- url: /static/pages
-  static_dir: static/pages
+- url: /static
+  static_dir: static
   secure: always
   http_headers:
     Cache-Control: 'public, max-age=600'
 - url: /robots.txt
-  static_files: static/pages/robots.txt
```

Comment:
Let's organize this file a bit -- how about putting this rule next to favicon.ico?